x-pack/plugin/esql/src/test/java/org/elasticsearch/xpack/esql/stats/VerifierMetricsTests.java

```diff
@@ -317,6 +341,8 @@ public void testRow() {
         assertEquals(0, drop(c));
         assertEquals(0, keep(c));
         assertEquals(0, rename(c));
+
+        assertNullFunction("EQUALS", c);
```

Comment:
This is an operator.

Reply:
Yes, if we rely on Function hierarchy we'll also get operators.
Probably we don't want it, or at least we want them to be two separate statistics.
I'll refactor it, filtering this information based on is registered in FunctionRegistry